projects/packages/search/src/dashboard/components/pages/upsell-page.jsx

```diff
@@ -69,24 +236,9 @@ export default function UpsellPage( { isLoading = false } ) {
 						a8cLogoHref={ AUTOMATTIC_WEBSITE }
 					>
 						<AdminSectionHero>
-							<Container horizontalSpacing={ 3 } horizontalGap={ 3 }>
-								<Col lg={ 6 } md={ 6 } sm={ 4 }>
-									<h1>{ __( 'The best WordPress search experience', 'jetpack-search-pkg' ) }</h1>
-									<SearchPromotionBlock />
-								</Col>
-								<Col lg={ 1 } md={ 1 } sm={ 0 } />
-								<Col lg={ 5 } md={ 6 } sm={ 4 }>
-									<PricingCard
-										ctaText={ __( 'Get Jetpack Search', 'jetpack-search-pkg' ) }
-										icon="data:image/svg+xml,%3Csvg width='32' height='32' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M21 19l-5.154-5.154C16.574 12.742 17 11.42 17 10c0-3.866-3.134-7-7-7s-7 3.134-7 7 3.134 7 7 7c1.42 0 2.742-.426 3.846-1.154L19 21l2-2zM5 10c0-2.757 2.243-5 5-5s5 2.243 5 5-2.243 5-5 5-5-2.243-5-5z' fill='%23000'/%3E%3C/svg%3E"
-										infoText={ priceAfter === priceBefore ? basicInfoText : onSaleInfoText }
-										// eslint-disable-next-line react/jsx-no-bind
-										onCtaClick={ sendToCart }
-										priceAfter={ priceAfter }
-										priceBefore={ priceBefore }
-										currencyCode={ priceCurrencyCode }
-										title={ __( 'Jetpack Search', 'jetpack-search-pkg' ) }
-									/>
+							<Container horizontalSpacing={ 8 }>
+								<Col lg={ 12 } md={ 12 } sm={ 12 }>
+									<ThemeProvider>{ pricingTable }</ThemeProvider>
 								</Col>
 							</Container>
```

Comment:
We don't want to remove the existing upsell interface until we launch the new pricing tiers. Instead, we'll want to conditionally render either the old or the new upsell screen using a feature gate.

To do this, I recommend using our dashboard's [built-in feature flag capability](https://github.com/Automattic/jetpack/blob/trunk/projects/packages/search/src/dashboard/class-initial-state.php#L85) which will look for a comma separated value stored in the `features` query string. For example, navigating to:

`/wp-admin/admin.php?page=jetpack-search&features=pricing,pricing2`

will store a value of `['pricing', 'pricing2']` in `window.JETPACK_SEARCH_DASHBOARD_INITIAL_STATE.features`.

I imagine our implementation might look something like this:
```
{ isNewPricingEnabled ? <NewUpsellContainer /> : <OldUpsellContainer /> }
```

Reply:
Addressed in [7017e1b](https://github.com/Automattic/jetpack/pull/26408/commits/7017e1ba4d1f5a47716bc6442ae78614c2052dfa)